app/model/JourneyConfigDefaults.scala

```diff
@@ -82,8 +82,8 @@ object JourneyConfigDefaults {
     val EDIT_PAGE_COUNTRY_LABEL = "Country"
     val EDIT_PAGE_SUBMIT_LABEL = "Continue"
 
-    val LOOKUP_PAGE_TITLE = "Find the address"
-    val LOOKUP_PAGE_HEADING = "Find the address"
+    val LOOKUP_PAGE_TITLE = "Find $UK address"
```

Comment:
val LOOKUP_PAGE_TITLE = s"Find $UK address"
  
This needs to be prefixed with s (for string interpolation), like above, otherwise it can't treat UK as a variable and substitute its value.